### PR TITLE
accu163 Beginning of work

### PR DESCRIPTION
**Pull request by TerebDan**

### CODE REVIEW

File: accu163/roles/APP_django/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+- name: Create Python virtual env and install packages inside the virtual env
+  pip:
+    name: 
+    - django==1.4 
+    - mysqlclient
+    virtualenv: /home/dan/venv
+    virtualenv_command: virtualenv
+    virtualenv_site_packages: no
+    virtualenv_python: python2
```
Comment (TerebDan) [inline accu163/roles/APP_django/tasks/main.yml:55]: There are no steps to start the server. The server is started manually.

File: accu163/group_vars/Deploy [hunk 2]
```diff
@@ -0,0 +1,3 @@
+---
+ansible_user                    : dan
+ansible_ssh_pass                : 1234
```
Comment (Neve) [inline accu163/group_vars/Deploy]: Do not store passwords in public repos, please use gitignore

File: accu163/roles/APP_django/tasks/main.yml [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    - curl 
+    - virtualenv 
+    - python2-dev 
+    - libmysqlclient-dev 
+    - gcc
+
+- name: Checking pip installed
+  stat:
+    path: /usr/bin/pip
+  register: result
```
Comment (Neve) [inline accu163/roles/APP_django/tasks/main.yml:17]: next time please be more specific in naming variables

File: accu163/roles/DB/.github/stale.yml [hunk 4]
```diff
@@ -0,0 +1,57 @@
+# Configuration for probot-stale - https://github.com/probot/stale
```
Comment (Neve) [inline accu163/roles/DB/.github/stale.yml]: Do we actually need this file in  the PR?

Comment (Neve) [reply accu163/roles/DB/.github/stale.yml]: please add .github folder to gitignore

File: accu163/roles/DB/molecule/default/converge.yml [hunk 5]
```diff
@@ -0,0 +1,16 @@
+---
```
Comment (Neve) [inline accu163/roles/DB/molecule/default/converge.yml]: either introduce molecule or remove tests from the playbook level

File: accu163/roles/LB/files/nginx.conf [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+    # Load modular configuration files from the /etc/nginx/conf.d directory.
+    # See http://nginx.org/en/docs/ngx_core_module.html#include
+    # for more information.
+    include /etc/nginx/conf.d/*.conf;
+
+    server {
+        listen       80;
+    
+        location / {
+            proxy_pass http://192.168.0.141:8000;
```
Comment (Neve) [inline accu163/roles/LB/files/nginx.conf]: please use variables instead of hardcode

Comment (Neve) [reply accu163/roles/LB/files/nginx.conf]: convert this config to j2 template, see my.cnf as example

File: accu163/roles/LB/meta/main.yml [hunk 7]
```diff
@@ -0,0 +1,52 @@
+galaxy_info:
```
Comment (Neve) [inline accu163/roles/LB/meta/main.yml]: consider removing since there is no plans for ansible galaxy

File: accu163/roles/LB/tasks/main.yml [hunk 8]
```diff
@@ -0,0 +1,43 @@
...
+  systemd:
+    name: firewalld
+    state: reloaded
+
+- name: Disable SELinux
+  ansible.posix.selinux:
+    state: disabled      
+
+- name: Remove nginx.conf
+  ansible.builtin.file:
```
Comment (Neve) [inline accu163/roles/LB/tasks/main.yml]: use template instead of replacing the file

File: accu163/roles/LB/tests/inventory [hunk 9]
```diff
@@ -0,0 +1,2 @@
+localhost
```
Comment (Neve) [inline accu163/roles/LB/tests/inventory]: consider removing

File: accu163/roles/LB/tests/test.yml [hunk 10]
```diff
@@ -0,0 +1,5 @@
+---
```
Comment (Neve) [inline accu163/roles/LB/tests/test.yml]: consider removing

**Review by Neve — CHANGES_REQUESTED** [review]
DB playbook overall:
if you are using community playbooks, please make sure to remove all unsupported platforms and features to keep codebase clean

**Review by Neve — APPROVED** [review]

**Pull request merged**